var/spack/repos/builtin/packages/g2c/package.py

```diff
@@ -64,6 +64,7 @@ class G2c(CMakePackage):
     depends_on("openjpeg", when="+openjpeg")
     depends_on("libxml2@2.9:", when="+build_v2_api")
 
+    conflicts("+utils ~build_v2_api", msg="+utils requires G2C API")
```

Comment:
Please change the default for these variants, so they're not in conflict by default. Having a default that is in a conflict will make it harder for clingo to decide what is optimal, since it needs to flip one of these two variants.

Reply:
Done. Thanks